x/ccv/provider/keeper/relay.go

```diff
@@ -70,6 +78,7 @@ func (k Keeper) HandleVSCMaturedPacket(ctx sdk.Context, chainID string, data ccv
 	for _, unbondingOp := range unbondingOps {
 		// remove consumer chain ID from unbonding op record
 		unbondingOp.UnbondingConsumerChains, _ = removeStringFromSlice(unbondingOp.UnbondingConsumerChains, chainID)
+		k.Logger(ctx).Debug("unbonding operation matured on consumer", "chainID", chainID, "opID", unbondingOp.Id)
 
 		// If unbonding op is completely unbonded from all relevant consumer chains
 		if len(unbondingOp.UnbondingConsumerChains) == 0 {
```

Comment:
Perhaps only log the case when `len(unbondingOp.UnbondingConsumerChains) == 0`, it might be too noisy otherwise.

Reply:
It's debug, it's okay to be noisy :)